octokit/resources.py

```diff
@@ -226,7 +196,5 @@ def fetch_resource(self, method, *args, **kwargs):
     response = self.session.send(prepared_req)
 
     schema = self.parse_schema(response)
-    self.rels = self.parse_rels(response)
-    self.session.last_response = response
 
-    return Resource(self.session, schema=schema, name=humanize(self.name))
+    return Resource(self.session, response=response, schema=schema, name=humanize(self.name))
```

Comment:
I think since we are passing the response in as parameter, we should just move the `self.parse_schema(response)` to the constructor